armi/bookkeeping/db/database3.py

```diff
@@ -680,10 +680,7 @@ def load(
         updateGlobalAssemNum : bool, optional
             DeprecationWarning: This is unused.
         updateMasterCs : bool, optional
-            Whether to apply the cs (whether provided as an argument or read from
-            the database) as the primary for the case. Default True. Can be useful
-            if you don't intend to use the loaded reactor as the basis for further
-            computations in the current operator.
+            TODO: Deprecated. Slated for removal.
```

Comment:
does removing `updateMasterCs` here cause a lot of additional work? Not sure if it should be a part of the master cs cleanup or if it should wait.

Reply:
I was trying to break the API in fewer ways, if possible. But I could look downstream and see how big a footprint adding this change in would make. Sure.

Reply:
I'm assuming it isn't altered from the default ever/often but if it is then don't worry about it!